static/src/javascripts/bootstraps/enhanced/article.js

```diff
@@ -31,6 +32,7 @@ const modules = {
         const mainColumn = qwery('.js-content-main-column');
         // only render when we have >1000px or more (enough space for ad + most popular)
         if (
+            !config.hasTone('Match reports') &&
```

Comment:
I was wondering if we should add a comment, 😅 the other option is for curious people to Git Blame and find this PR... 🙃 

I'm not opposed to checking the height, but wonder how expensive it would be to check every time? 

We could only check the article height if page is Match Report?
`if(config.hasTone('Match reports') && mainColumn[0].offsetHeight > 1700)`